docs/source/android-section.md

```diff
@@ -0,0 +1,57 @@
+# Android
+
+## Quick Start & Integration
+[Getting started with ExecuTorch on Android](using-executorch-android)
```

Comment:
When I click on it from here

https://docs-preview.pytorch.org/pytorch/executorch/14720/android-section.html

the left navigation disappears 

https://docs-preview.pytorch.org/pytorch/executorch/14720/using-executorch-android.html